frontend/src/Components/NicknameBar.jsx

```diff
@@ -0,0 +1,21 @@
+import  React, {useState} from 'react';
+
+const NicknameBar = () => {
+    const [data, setData] = useState(null);
+
+    const handleName = (Name) => {
+        setData(Name.target.value)
+    }
+
+    return (
+        /* supposed to save the user nickname to the database */
+        /* ENTER button appear when is entered, onClick to save the 
+            nickname */
+        <div>
+            <input placeholder="Enter Nickname" type= "text" onChange = {handleName}></input>
```

Comment:
Probably isnt an issue but I dont think we have a database at all for first iteration

Reply:
Agree, idk how are we gonna keep and display the nickname for different player so just comment for future improvement